crud.py

```diff
@@ -0,0 +1,71 @@
+from sqlalchemy.orm import Session
+
+from db import models
+import schemas
+
+
+def get_book_list(db: Session, author_id: str | None = None):
+    queryset = db.query(models.DBBook)
+
+    if author_id:
+        queryset = queryset.filter(models.DBBook.author_id == author_id)
+
+    return queryset
+
+
+def get_book(db: Session, book_id: int):
```

Comment:
This function is fine but make sure to handle cases where the book might not be found.